src/parser/contexts.h

```diff
@@ -1687,9 +1694,17 @@ struct ParseDefsCtx : TypeParserCtx<ParseDefsCtx> {
     return wasm.memories[0]->name;
   }
 
-  void setSrcLoc(const Annotation& annotation) {
-    assert(annotation.kind == srcAnnotationKind);
-    Lexer lexer(annotation.contents);
+  void setSrcLoc(const std::vector<Annotation>& annotations) {
+    const Annotation* annotation = nullptr;
+    for (auto& a : annotations) {
+      if (a.kind == srcAnnotationKind) {
+        annotation = &a;
```

Comment:
This is just moved code, but I a question: can there be more than one `srcAnnotationKind`? (seems like if so then the last wins?)

Reply:
Yes and yes. Is there some other existing behavior I should try to match instead?

Reply:
I can't think of one, so lgtm.